tests/manage/z_cluster/cluster_expansion/test_add_capacity.py

```diff
@@ -18,16 +19,36 @@
 )
 from ocs_ci.ocs import constants
 from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.resources.pod import get_osd_pods
 from ocs_ci.ocs.resources import storage_cluster
 from ocs_ci.utility.utils import ceph_health_check
 from ocs_ci.ocs.cluster import CephCluster
 from ocs_ci.ocs.resources.storage_cluster import osd_encryption_verification
 from ocs_ci.framework.pytest_customization.marks import skipif_openshift_dedicated
 
 
+logger = logging.getLogger(__name__)
+
+
 def add_capacity_test():
     osd_size = storage_cluster.get_osd_size()
+    existing_osd_pods = get_osd_pods()
+    existing_osd_pod_names = [pod.name for pod in existing_osd_pods]
     result = storage_cluster.add_capacity(osd_size)
+    osd_pods_post_expansion = get_osd_pods()
```

Comment:
There is a generic function to get the restart count for all the pods in a given namespace in [here](https://github.com/red-hat-storage/ocs-ci/blob/048dd21117f968becea32c15144cf7f5ce26035f/ocs_ci/ocs/resources/pod.py#L1531). We could use that for not just checking the restarts of osd pod but also other pods...
Usage of the above function can be seen [here](https://github.com/red-hat-storage/ocs-ci/blob/master/tests/manage/z_cluster/cluster_expansion/test_add_capacity_entry_exit_criteria.py#L212)

Reply:
restart count is not what we are looking for. We are looking for OSD pods deletion

Reply:
I updated the logging with this information. thanks for noticing